x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_stat.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiStat, EuiToolTip } from '@elastic/eui';
+
+interface Props {
+  isLoading: boolean;
+  title: number | string;
+  isMSE: boolean;
+}
+
+export const EvaluateStat: FC<Props> = ({ isLoading, isMSE, title }) => {
+  const meanSquaredErrorText = i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.meanSquaredErrorText',
+    {
+      defaultMessage: 'Mean squared error',
+    }
+  );
+  const rSquaredText = i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.rSquaredText',
+    {
+      defaultMessage: 'R squared',
+    }
+  );
+  const meanSquaredErrorTooltipContent = i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.meanSquaredErrorTooltipContent',
+    {
+      defaultMessage:
+        'Measures how well the regression analysis model is performing. Avg. squared sum of the difference between true and predicted values.',
+    }
+  );
+  const rSquaredTooltipContent = i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.rSquaredTooltipContent',
+    {
+      defaultMessage:
+        'Represents the goodness of fit. Measures how well the observed outcomes are replicated by the model.',
+    }
+  );
```

Comment:
Nit: Suggest to move those messages out of the component in the files main scope so they don't have to be recreated on every render.

Reply:
Good catch! Updated in 494ac87820e7fc2863e03dc244f44c9a290bff4c